tests/framework_graphql/test_application.py

```diff
@@ -414,10 +443,12 @@ def test_operation_metrics_and_attrs(target_application):
         "graphql.operation.name": "MyQuery",
     }
 
-    # Span count 16: Transaction, Operation, and 7 Resolvers and Resolver functions
+    # Span count 16 for v3.2: Transaction, Operation, and 7 Resolvers and Resolver functions
+    # Span count 15 for v3.2: Transaction, Operation, and 6 Resolvers and Resolver functions
```

Comment:
Unless they're optimizing away some resolvers, I think we have an issue with span count as well